pkg/addon/templateagent/registration.go

```diff
@@ -25,13 +27,29 @@ import (
 )
 
 const (
-	TLSCACert = "ca.crt"
-	TLSCAKey  = "ca.key"
 	// AddonTemplateLabelKey is the label key to set addon template name. It is to set the resources on the hub relating
 	// to an addon template
 	AddonTemplateLabelKey = "open-cluster-management.io/addon-template-name"
```

Comment:
TODO move this to the api repo.

Reply:
Because we restricted the sign ca secret type to `kubernetes.io/tls`, these are not needed.